frontend/src/components/podcast/podcast.tsx

```diff
@@ -68,11 +70,13 @@ const Podcast: React.FC = () => {
             )}
             <div className={styles.podcastInfoWrapper}>
               {isOwner && (
+                <Link
+                  to={`${AppRoute.PODCASTS_EDIT}/${podcast.id}`}
+                  className={styles.editLink}
+                />
+              )}
+              {(isOwner || isMaster) && (
```

Comment:
```suggestion
              {isAllowDelete && (
```
admin can **only** delete